bin/start

```diff
@@ -30,18 +32,22 @@ EOF
 }
 
 function runScriptWithSummon() {
-    summon --environment $SUMMON_ENV -f ./summon/secrets.yml $1
+    summon --environment $SUMMON_ENV -f $1 $2
 }
```

Comment:
let's add variables to describe what 1 and 2 are, so it is more clear to the reader

something like:
```
secrets_file=$1
command=$2
summon --environment $SUMMON_ENV -f $secrets_file $command
```

Reply:
Because i updated directories, I reverted it back to:
```
function runScriptWithSummon() {
    summon --environment $SUMMON_ENV -f ./summon/secrets.yml $1
}
```
IMO i think it is pretty straightforward now and think command is a little bit redundant. What do you think?

Reply:
i would still have it because now you need to read the whole line to see that this function needs an input which can be missed.

Also - unrelated but while you're at it, you can remove the word `function` from all the functions. it is not needed. it's ok if you prefer to not do it as part of this PR.